core/templates/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -568,6 +568,7 @@ angular.module('oppia').directive('stateResponses', [
             var activeStateName = $scope.stateName;
             return outcome && (outcome.dest === activeStateName);
           };
+
```

Comment:
Please check the Files Changed tab for your PR and remove all changes to files which do not need to be changed.

Reply:
done